roles/nfd_operator_deploy_custom_commit/vars/main/resources.yml

```diff
@@ -0,0 +1,13 @@
+---
+# Custom label to apply to the node where we'll load fuse
+tuned_module_fuse_label: tuned.module.fuse
+# NFD Deploy Script
+nfd_install_script: roles/nfd_operator_deploy_custom_commit/scripts/deploy_operator.sh
+# Manifest for installing the nfd operator from a custom commit
+tuned_module_fuse: roles/nfd_operator_deploy_custom_commit/files/tuned_module_fuse.yml
```

Comment:
what about pointing to `roles/gpu_operator_deploy_custom_commit/files/tuned_module_fuse.yml`?

could also move `tuned_module_fuse_label` and `tuned_module_fuse` into ` roles/nfd_operator_deploy_custom_commit
/vars/main/fuse.yml ` and symlink it into this role?

Reply:
I think that is a good TODO for a following PR, to reuse vars from many roles